openhtf/core/test_executor.py

```diff
@@ -18,8 +18,10 @@
 import sys
 import threading
 
+import openhtf
```

Comment:
Don't import the base openhtf module; we are trying to remove the circular imports.

Reply:
Done